src/Containers/Note/Note.scss

```diff
@@ -0,0 +1,53 @@
+h3 {
+  font-family: "Open Sans", sans-serif;
```

Comment:
We could add the imported font and some of the main colors to the _variables.scss file

Reply:
Totally agree, this will be good to refactor.